rivertest/worker_test.go

```diff
@@ -58,48 +80,43 @@ func TestWorker_Work(t *testing.T) {
 
 			return nil
 		})
-		tw := NewWorker(t, driver, config, worker)
-		require.NoError(t, tw.Work(context.Background(), t, testArgs{Value: "test"}, nil))
+		tw := NewWorker(t, bundle.driver, bundle.config, worker)
+		res, err := tw.Work(context.Background(), t, bundle.tx, testArgs{Value: "test"}, nil)
```

Comment:
Maybe replace this `context.Background()` with `ctx` since it's broadly defined now, and it cleans up a lot of visual noise.

Reply:
Done, did that in the whole file 👍